### ISSUE TITLE
How to install on a Pi?

### ISSUE THREAD

**Issue by YouveGotMeowxy** [DockStation/dockstation#232]
I have a Raspberry Pi 3 B+ and I want to install DS on it; I'm new to Linux and may just be missing something easy but, I need a .deb file for the Pi and the only .deb I see available in the Releases is for AMD, and I believe I need one for armhf.

How can I install?

Thanks!
Kacey

**Comment by jamezrin** [comment]
I don't think there is support for ARM, but I don't think it should support ARM, or at least the Pi. You are better off using something more lightweight, like the console, or a web based manager.

**Comment by YouveGotMeowxy** [comment]
@jamezrin I do use Portainer to connect to the Pi, but I also need a local client at times. I do the same with all of my machines, Pi or otherwise; I use portainer mainly, and something like DS or Kitematic when also needed.

50-60mbs isn't too heavy when you run the Pi with a 64gb sd card.

**Comment by jamezrin** [comment]
Yeah, the package is lightweight, but in runtime the process itself isn't. It consumes a bunch of memory and CPU.

You can use DockStation on your computer and connect to the remote docker instance, that should work better.

**Comment by igor-lemon** [comment]
Unfortunately nope. We support just PC/Mac
So, we realized our disadvantages and we want to make a really cross-platform system a mix of current DockStation and Portainer + mobile devices support. It means that you will be able to manage Docker from any device to any device that supportsDocker.

**Comment by YouveGotMeowxy** [comment]
@jamezrin oh ok, I see what you're saying, sorry I misunderstood. I thought you just meant the package size alone.

I'd like to try connecting to the Pi's Docker daemon with DS running on a different computer, but I can't seem to figure out the exact settings to do so. Can you help me out? I'd really appreciate it. Like a screenshot of the settings to use, exactly. I'm not particularly good with these things.

**Comment by Rathna-K** [comment]
Trying to follow this thread, so can dock station be installed in Pi or no? And if cannot be installed then can it be installed in Windows and then manage the Pi containers?

**Comment by FearlessNA** [comment]
> Trying to follow this thread, so can dock station be installed in Pi or no? And if cannot be installed then can it be installed in Windows and then manage the Pi containers?

Yes,
Install on your Windows PC for example, then connect to your Pi's IP via the Connections Manager by just clicking the + button saying ADD.
